modApiServer/src/org/aion/api/server/rpc3/Web3EntryPoint.java

```diff
@@ -50,6 +51,11 @@ public String call(String requestString){
         RpcError err = null;
         Integer id = null;
         Object resultUnion = null;
+        Stopwatch stopwatch = null;
+        if (logger.isDebugEnabled()){
```

Comment:
If the logger got pass from the AionLogger, then doesn't need this check anymore.

Reply:
We are using the check to avoid creating an instance of the stopwatch if it is not necessary.

Reply:
I mean inside the AionLogger, it already checked the isDebugEnabled() call. So you don't have to check in the code if this is an AionLogger.